.cirrus.yml

```diff
@@ -89,6 +73,64 @@ task:
   env:
     << : *CIRRUS_EPHEMERAL_WORKER_TEMPLATE_ENV
 
+task:
+  name: "Win64 [unit tests, no functional tests] [msvc]"
+  windows_container:
+    cpu: 8
```

Comment:
why 8 when the average usage is 2?

Reply:
With `cpu: 4`:
 
![Screenshot from 2021-09-07 06-45-03](https://user-images.githubusercontent.com/32963518/132281199-c3c4f8d3-dd99-4b50-bab7-60d58a99af49.png)

And the total time with a populated `vcpkg` cache:

![Screenshot from 2021-09-07 06-46-09](https://user-images.githubusercontent.com/32963518/132281269-f262a40b-a91a-471a-8a5e-de59e59cf590.png)